Godeps/_workspace/src/k8s.io/kubernetes/pkg/admission/plugins.go

```diff
@@ -36,8 +38,29 @@ type Factory func(client clientset.Interface, config io.Reader) (Interface, erro
 var (
 	pluginsMutex sync.Mutex
 	plugins      = make(map[string]Factory)
+
+	defaultOnPlugins  = sets.String{}
+	defaultOffPlugins = sets.String{}
+
+	PluginEnabledFn = func(config io.Reader, defaultValue bool) (bool, error) {
+		return defaultValue, nil
+	}
 )
 
+type PluginEnabledFunc func(config io.Reader, defaultValue bool) (bool, error)
+
+func AddDefaultOnPlugins(names ...string) {
```

Comment:
this feels like it can be local to the object that provides the `PluginEnabledFn`, remove this from upstream commit.